src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -597,7 +602,7 @@ export class PatientViewPageStore {
                     };
                 });
 
-            if (this.geneticProfileIdDiscrete.result) {
+            if (this.geneticProfileIdDiscrete.result && copyNumberCountIdentifiers.length > 0) {

```

Comment:
I'm pretty sure I deleted `copyNumberCountData` with the PR to make it lazy loaded